package.json

```diff
@@ -38,48 +40,58 @@
     "ie 11"
   ],
   "peerDependencies": {
-    "vue": "^2.5.x",
-    "@mdi/font": "^3.0.39"
+    "vue": "^2.6.10",
+    "@mdi/font": "^4.4.95"
```

Comment:
Do we want to bump these? Do we have a case for needing the latest `vue` and `mdi`? This is forcing consumers to bump their libs and if we don't need it I wouldn't do it just for the sake of it.

Reply:
I don't see a reason why we wouldn't bump them. There are no breaking changes on neither of those packages, backwards compatibility assured. Also this will be in sync with @ExtensionEngine/tailor https://github.com/ExtensionEngine/tailor/pull/350

Reply:
While that might be true it still doesn't change the fact that we'll be forcing people to bump with no reason what so ever.
Also, peer dependency should represent the minimum that we rely on and current ones are fine.
@vladimyr I'm interested in your opinion on this?

Reply:
My reasoning is that we want our codebase to be up to date whenever possible. Also, forcing others to update and be up to date is a good thing. Win win situation. 

> Also, peer dependency should represent the minimum that we rely on and current ones are fine.

Correct, and our bare minimum now are the latest versions. 💋

Reply:
But if you insist and want to stop progress, I'll revert.

Reply:
Reverted.

Reply:
> While that might be true it still doesn't change the fact that we'll be forcing people to bump with no reason what so ever.
> Also, peer dependency should represent the minimum that we rely on and current ones are fine.

I'm partially with @kronicker on this one because bumping Vue might be hard investment to sell. OTOH `@mdi/font` is something that should be no brainer updating and more that I think about it more it seems that having it declared as:
```
"@mdi/font": "*"
```
is right way to go. We don't directly rely on it and whatever version works for consumers works for us too?

Reply:
I'll keep it reverted. Please consider merging this one.